base/src/test/java/org/aya/concrete/DesugarTest.java

```diff
@@ -54,6 +54,16 @@ public class DesugarTest {
       }""");
   }
 
+  @Test public void arrays() {
+    desugarAndPretty("""
+    def simpleList => [ 1, 2, 3 ]
+    def listGenerator => [ x * y | x <- xs, y <- ys ]
+    """, """
+    def simpleList => :< 1 (:< 2 (:< 3 nil))
+    def listGenerator => >>= xs (\\ (x : _) => >>= ys (\\ (y : _) => pure (x * y)))
+    """);   // Aya doesn't know the `:<` and `>>=` are infix function in this case, how to let she know?
```

Comment:
I think you have to define those functions. Write something like `def infix >>= => {??}` I guess